nucliadb/nucliadb/writer/run.py

```diff
@@ -18,8 +18,8 @@
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
 
+from nucliadb.reader.app import get_application
```

Comment:
you're calling get_application from reader in the writer component!

Reply:
haha no wonder why tests were not working